gcs/conf/gcs-core-default.xml

```diff
@@ -572,6 +572,15 @@
     </description>
   </property>
 
+  <property>
+    <name>fs.gs.inputstream.support.content.encoding.enable</name>
```

Comment:
Maybe make this more explicitly about gzip since it's not really about supporting general content encoding?

Reply:
Done.